src/y2start/y2start

```diff
@@ -39,6 +39,7 @@ if !Yast::WFM.ClientExists(args[:client_name])
 end
 
 
+NO_CLI_CLIENTS = ["installation", "view_anymsg"]
```

Comment:
possible..that reminds me that rubocop is not set to this file as it does not have .rb suffix